payroll/gql_queries.py

```diff
@@ -141,3 +141,22 @@ class Meta:
             "version": ["exact"],
         }
         connection_class = ExtendedConnection
+
+
+class CsvReconciliationUploadGQLType(DjangoObjectType):
+    uuid = graphene.String(source='uuid')
+
+    class Meta:
+        model = CsvReconciliationUpload
+        interfaces = (graphene.relay.Node,)
+
+    filter_fields = {
```

Comment:
indentation issue. Should be
`    class Meta:
        model = CsvReconciliationUpload
        interfaces = (graphene.relay.Node,)

        filter_fields = {
            "id": ["exact"],
            "date_created": ["exact", "lt", "lte", "gt", "gte"],
            "date_updated": ["exact", "lt", "lte", "gt", "gte"],
            "status": ["exact", "iexact", "istartswith", "icontains"],
            "is_deleted": ["exact"],
            "version": ["exact"],
            **prefix_filterset("payroll__", PayrollGQLType._meta.filter_fields),
        }
        connection_class = ExtendedConnection`

Reply:
Fixed